docs/http/Static-Files.md

```
@@ -0,0 +1,737 @@
+  );
+  
+  // Serve files from 'public' directory
+  app.useStaticAssets('public');
+  
+  await app.listen(3000);
+}
+bootstrap();
+```
+
+**Directory structure:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)